README.rst

```diff
@@ -1,148 +1,186 @@
 FIWARE Policy Manager GE - Facts
-____________________
+________________________________
```

Comment:
You can use for main titles:

       ==========
       Main title
       ==========

You should add under the main section a brief description about the content of this repo and the context of it. Take a look at the REAMDE of [FiHealth project] (https://github.com/telefonicaid/fiware-health/blob/develop/fiware-region-sanity-tests/README.rst): Take a look at the REAMDE of [FiHealth project] (https://github.com/telefonicaid/fiware-health/blob/develop/fiware-region-sanity-tests/README.rst):